### DISCUSSION TITLE
Remove bound box in p.add_mesh_slice function

### DISCUSSION THREAD

**Discussion by SantiagoPendon** [pyvista/pyvista#5346] [Q&A] [answered]
Hi to everyone once again :)

Following the next tutorial https://docs.pyvista.org/version/stable/examples/03-widgets/plane-widget.html#sphx-glr-examples-03-widgets-plane-widget-py
I'm very impressed with the results, but I would like to know if is possible remove the black box cube. The arrow to displace the volume and slicer is great, but the black box is unnecessary for my intentions. 

Thanks so much!

**Comment by tkoyama010** [comment, marked as answer]
You can set the color opacity to 0.0 and remove the boundary.
```python
import pyvista as pv
from pyvista import examples

mesh = examples.download_carotid()

p = pv.Plotter()
p.add_mesh(mesh.contour(8).extract_largest(), opacity=0.5)


def my_plane_func(normal, origin):
    slc = mesh.slice(normal=normal, origin=origin)
    arrows = slc.glyph(orient='vectors', scale="scalars", factor=0.01)
    p.add_mesh(arrows, name='arrows')

# Set opacity of color 0.0
color=pv.Color("black", opacity=0.0)

p.add_plane_widget(my_plane_func, color=color)
p.show_grid()
p.add_axes()
p.show()
```
![image](https://github.com/pyvista/pyvista/assets/7513610/cffce790-30b1-4fc3-a696-ae6500727a63)

**Reply by tkoyama010** [reply]
For this, it would be more convenient to be able to specify directly whether to hide them, so we will add this functionality. Thanks for the inspiration.

**Reply by SantiagoPendon** [reply]
Hi tkoyama010: The solution is awesome but we are interest in `add_mesh_slice()` function.

It's also possible make this trick in  this function?

**Reply by tkoyama010** [reply]
The trick is the same but we need to use `widget_color` args for this not `color`.
```python
import pyvista as pv
from pyvista import examples

vol = examples.download_brain()

# Set opacity of color 0.0
widget_color=pv.Color("black", opacity=0.0)

p = pv.Plotter()
p.add_mesh_slice(vol, widget_color=widget_color)
p.show()
```
![image](https://github.com/pyvista/pyvista/assets/7513610/903c779b-f347-47fd-801b-67a5f13a26ac)

**Reply by SantiagoPendon** [reply]
That's the key!!!! Thanks a lot :)

One more thing:

`import pyvista as pv
from pyvista import examples

vol = examples.download_brain()

p = pv.Plotter()
p.add_mesh_slice(vol)
p.clear_plane_widgets()
p.show()`

The line `p.clear_plane_widgets()` should clear the scene and a clear scene, but the line crashes and restarts my kernel. 

My version is:

--------------------------------------------------------------------------------
  Date: Fri Dec 15 13:23:11 2023 Hora estándar romance

                OS : Windows
            CPU(s) : 12
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.7 GiB
       Environment : IPython
        GPU Vendor : NVIDIA Corporation
      GPU Renderer : NVIDIA T400/PCIe/SSE2
       GPU Version : 4.5.0 NVIDIA 516.94
  MathText Support : False

  Python 3.11.5 | packaged by Anaconda, Inc. | (main, Sep 11 2023, 13:26:23)
  [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.42.3
               vtk : 9.3.0
             numpy : 1.24.3
        matplotlib : 3.7.2
            scooby : 0.9.2
             pooch : 1.8.0
            pillow : 9.4.0
           imageio : 2.26.0
         pyvistaqt : 0.11.0
             PyQt5 : 5.15.7
           IPython : 8.15.0
          colorcet : 3.0.1
        ipywidgets : 8.0.4
             scipy : 1.11.1
              tqdm : 4.65.0
        jupyterlab : 3.6.3
      nest_asyncio : 1.5.6

  Intel(R) oneAPI Math Kernel Library Version 2023.1-Product Build 20230303
  for Intel(R) 64 architecture applications
--------------------

Any suggestion about that? Your help is so great :)

**Reply by tkoyama010** [reply]
@SantiagoPendon I am currently unable to do the research, so could you please create a new Discussion? I would like to separate threads to make this forum more searchable. It makes other community members notice your question.
